Modix/Modules/NoteModule.cs

```diff
@@ -0,0 +1,172 @@
+﻿namespace Modix.Modules
+{
+    using System;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    using Discord;
+    using Discord.Commands;
+    using Discord.WebSocket;
+    using Modix.Data.Models;
+    using Modix.Data.Models.Moderation;
+    using Modix.Services.Moderation;
+    using Serilog;
+    using Services.CommandHelp;
+    using Tababular;
+
+    [Name("Add/Remove/Search Admin notes"), RequireUserPermission(GuildPermission.Administrator), HiddenFromHelp]
+    public class NoteModule : ModuleBase
+    {
+        private readonly IModerationService _moderationService;
+
+        public NoteModule(IModerationService moderationService)
+        {
+            _moderationService = moderationService;
+        }
+
+        [Command("note add"), Summary("Adds a note to a user")]
+        public async Task CreateNote(IGuildUser user, [Remainder] string message)
+        {
+            var guildUser = Context.User as SocketGuildUser;
+
+            // TODO: We shouldn't need to do this once claims are working
+            if (!IsStaff(guildUser) && !IsOperator(guildUser))
+            {
```

Comment:
Can these checks be moved into a PreconditionAttribute implementation?

Reply:
Maybe but I don't know what that means.

Reply:
Claims auth should make this unnecessary at some point, so I don't know that it's worth the effort if the check is just going to be removed.

Reply:
Claims auth should work NOW, after the rebase. Drop those methods and checks.